app/code/Magento/CacheInvalidate/Model/PurgeCache.php

```diff
@@ -131,10 +132,27 @@ private function sendPurgeRequestToServers($socketAdapter, $servers, $formattedT
                 $socketAdapter->read();
                 $socketAdapter->close();
             } catch (\Exception $e) {
-                $this->logger->critical($e->getMessage(), compact('server', 'formattedTagsChunk'));
-                return false;
+                $unresponsiveServerError[] = "Cache host: " . $server->getHost() . ":" . $server->getPort() .
+                    "resulted in error message: " . $e->getMessage();
+                continue;
             }
         }
+
+        $errorCount = count($unresponsiveServerError);
+
+        if ($errorCount > 0) {
+
+            $loggerMessage = implode(" ", $unresponsiveServerError);
+
+            if ($errorCount == count($servers)) {
+                $this->logger->critical('No cache server(s) could be purged ' . $loggerMessage, compact('server',
```

Comment:
Each parameter should be in line by itself in multiline function comment, example:

    $this->logger->critical(
        'No cache server(s) could be purged ' . $loggerMessage,
        compact('server', 'formattedTagsChunk')
    );

Reply:
Sorry I think I misunderstood you :) It is changed now